austin_parham/function_basic_1.py

```diff
@@ -0,0 +1,103 @@
+def a():
+    return 5
+print(a())   # Prediction = 5
+
+def a():
+    return 5
+print(a()+a()) # Prediction = 10
+
+def a():
+    return 5
+    return 10
+print(a())   # Prediction = 10, Answer = 5 -- once a function sees a return it leaves the function entirely.
+
+def a():
+    return 5
+    print(10)
+print(a()) # Prediction = 5
+
+def a():
+    print(5)
+x = a()
```

Comment:
the function is called. Since there is no return, it returns back None. Therefore x = None